common/djangoapps/third_party_auth/management/commands/tests/test_saml.py

```diff
@@ -72,6 +82,43 @@ def setUp(self):
             metadata_source='https://www.testshib.org/metadata/testshib-providers.xml',
         )
 
+    def _setup_test_configs(self):
+        """
+        Helper method to create SAML configurations for fix-references tests.
+        Returns tuple of (old_config, new_config, provider_config)
```

Comment:
Docstrings should begin with single line summary.
```suggestion
        Helper method to create SAML configurations for fix-references tests.

        Returns tuple of (old_config, new_config, provider_config)
```

Reply:
Resolved